Backoffice/Form/Type/Component/NodeChoiceType.php

```diff
@@ -37,11 +38,30 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults(
             array(
                 'choices' => $this->getChoices(),
-                'attr' => array(
-                    'class' => 'orchestra-node-choice'
-                )
             )
         );
+
+        /*
+         * Normalize 'attr' option to force presence of 'orchestra-node-choice' html class.
+         * This class is mandatory to trigger execution of following coffee function on field :
+         * OpenOrchestra.FormBehavior.NodeChoice.activateBehaviorOnElements().
+         */
+        $resolver->setNormalizer('attr', function (Options $options, $value) {
+            if (!is_array($value)) {
+                $value = array();
+            }
+
+            $classList = array();
+            if (array_key_exists('class', $value) && !empty($value['class'])) {
+                $classList = explode(' ', $value['class']);
+            }
+            if (!in_array('orchestra-node-choice', $classList)) {
+                $classList[] = 'orchestra-node-choice';
+            }
+            $value['class'] = implode(' ', $classList);
+
+            return $value;
+        });
```

Comment:
You coud use the pattern on new_attr as in https://github.com/open-orchestra/open-orchestra-cms-bundle/blob/master/Backoffice/Form/Type/Component/KeywordsChoiceType.php